docker_config_generator/templates/python-poetry.Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.10 as python-base
+FROM python:3.10-bullseye as python-base
```

Comment:
`python:3.11-bullseye` would be conflict-less Ig, w.r.t PR #58

Reply:
Ok 👌we can update that